addons/io/org.openhab.io.neeo/src/main/java/org/openhab/io/neeo/internal/discovery/package-info.java

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
Darn - was hoping I could avoid slapping the attribute on every class by using package-info - but I'll change it around to avoid package-info classes.